pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig_test.go

```diff
@@ -33,6 +33,27 @@ const hugepagesAllocationService = `
         name: hugepages-allocation-1048576kB-NUMA0.service
 `
 
+const offlineCPUS = `
+      - contents: |
+          [Unit]
+          Description=OfflineCpus - offlined cpus: 6,7
+          Before=kubelet.service
+
+          [Service]
+          Environment=OFFLINE_CPUS=6,7
+          Type=oneshot
+          RemainAfterExit=true
+          ExecStart=/usr/local/bin/add-offline-cpus.sh
```

Comment:
need updates (add -> set)
probably also in other places

Reply:
done